hw/formal/README.md

```diff
@@ -311,6 +311,19 @@ For example:
           $past(rv_plic.le[src_sel]) || $past(intr_src_i[src_sel]), clk_i, !rst_ni)
 ```
 
+## Implementation Guidelines
+Assertions are widely used in RTL and FPV files. Below decumented the
+recommended guidelines in terms of where to implement assertions.
+* Basic assertions are recommended to implement directly under the RTL file.
+  This is commonly used for designers to write basic functional assertions for
+  sanity check
+* Assertions used for testbench to achieve verification goals are recommended to
```

Comment:
"Assertions used for the testbench to achieve verification goals should be implemented under the `ip/hw/module_name/fpv/vip` folder. This FPV environment can be automatically generated by the [`fpvgen.py` script](../../util/fpvgen/README.md)"